src/index.html

```diff
@@ -5,8 +5,434 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog">
+    <header class="header catalog__header">
+      <a href="#home" class="header__logo">
+        <img
+          src="./images/logo.png"
+          class="header__img"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover">laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="cards catalog__main">
+      <article class="card card--width cards__card" data-qa="card">
+        <div class="card__image-container">
+          <img src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot;
+            Retina 5K Monoblock (MNED2UA/A)"
+            class="card__image"
+          >
+        </div>
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="stars card__stars">
+          <div class="stars__display stars__display--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="stars__review">
```

Comment:
```suggestion
          <p class="card__review">
```
It should be an element of card, not stars